kernel/include/feeds.h

```diff
@@ -25,15 +30,15 @@ extern "C" {
 typedef enum osquery_event {
   END_OF_BUFFER_EVENT = 0,  // Null event used to signal the end of the buffer.
 
-#ifdef DEBUG
+#ifdef KERNEL_TEST
```

Comment:
Would you mind adding some inline docs/comments for what `KERNEL_TEST` is used for?